extensions/shared/components/style.scss

```diff
@@ -0,0 +1,23 @@
+@import '../styles/gutenberg-colors.scss';
+@import '../styles/gutenberg-variables.scss';
+
+// Styling copied from Gutenberg's `@wordpress/block-editor` styling for the Warning component
+// so we can use it on the frontend.
+.block-editor-warning {
+	border: $border-width solid $light-gray-500;
+	padding: 10px 14px;
+
+	.block-editor-warning__message {
+		line-height: $default-line-height;
+		font-family: $default-font;
+		font-size: $default-font-size;
+	}
+
+	.block-editor-warning__actions {
+		.components-button {
+			font-family: $default-font;
+			font-weight: inherit;
+			text-decoration: none;
```

Comment:
Annoyingly Storefront theme still leaks in this CSS:
```css
.hentry .entry-content a:not(.button) {
  text-decoration: underline;
}
```
<img width="220" alt="storefront" src="https://user-images.githubusercontent.com/87168/61950788-9c706b80-afb7-11e9-8a40-eea148b89b04.png">

Not a blocker.

Reply:
Fixed upstream: https://github.com/woocommerce/storefront/pull/1187